### PR TITLE
Docs: Add log topics all (pseudo), license, libiresearch. Remove compactor, datafiles.

### PR DESCRIPTION
**Pull request by Simran-B**
### Scope & Purpose

Add new log topics except for `rep-state` and `replication2`, which are likely internal only at this stage.

`libiresearch` is more of an internal one too but came up in a customer ticket.

The removed topics were probably MMFiles leftovers.

Would be nice to actually describe what each log topic is about...

- [x] :book: Documentation

### Checklist

- [x] Backports
  - [x] Backport for 3.9: #15767
  - [x] Backport for 3.8: #15768
  - [x] Backport for 3.7: #15769

#### Related Information

- [x] Docs PR: Adding libiresearch closes https://github.com/arangodb/docs/issues/417

### CODE REVIEW

File: Documentation/DocuBlocks/Rest/Administration/get_admin_log.md [hunk 1]
```diff
@@ -218,176 +218,179 @@ the API is disabled, all requests will be responded to with HTTP 403. If the
...
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{clustercomm,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
-@RESTBODYPARAM{collector,string,optional,string}
-One of the possible log levels.
+@RESTBODYPARAM{collector,string,optional,string} !
```
Comment (ansoboleva) [inline Documentation/DocuBlocks/Rest/Administration/get_admin_log.md:284]: what's this exclamation mark about?

File: Documentation/DocuBlocks/Rest/Administration/get_admin_log.md [hunk 2]
```diff
@@ -218,176 +218,179 @@ the API is disabled, all requests will be responded to with HTTP 403. If the
...
 
 @RESTBODYPARAM{httpclient,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{ldap,string,optional,string}
-One of the possible log levels (_Enterprise Edition only_).
+One of the possible log topics (_Enterprise Edition only_).
+
+@RESTBODYPARAM{libiresearch,string,optional,string}
```
Comment (ansoboleva) [inline Documentation/DocuBlocks/Rest/Administration/get_admin_log.md:323]: not 100% sure this should be present here

File: Documentation/DocuBlocks/Rest/Administration/get_admin_log.md (hunk 1 above)
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Administration/get_admin_log.md:284]: Good catch, not sure how that happened

```suggestion
@RESTBODYPARAM{collector,string,optional,string}
```

File: Documentation/DocuBlocks/Rest/Administration/get_admin_log.md (hunk 2 above)
Comment (Simran-B) [reply Documentation/DocuBlocks/Rest/Administration/get_admin_log.md:323]: File system-related ArangoSearch problems are logged under this topic and we suggested to adjust the level e.g. in https://github.com/arangodb/arangodb/issues/12628#issuecomment-697927571

Comment (ansoboleva) [reply Documentation/DocuBlocks/Rest/Administration/get_admin_log.md:323]: ok!

**Review by ansoboleva — APPROVED** [review]
reviewed

**Review by cpjulia — APPROVED** [review]
LGTM

**Comment by KVS85** [comment]
- [ ] https://jenkins.arangodb.biz:3456/view/Documentation/job/arangodb-ANY-examples/1724/

**Comment by KVS85** [comment]
- [ ] https://jenkins.arangodb.biz:3456/view/Documentation/job/arangodb-ANY-examples/1729/

**Pull request merged**